pkg/distributor/push_test.go

```diff
@@ -850,9 +817,7 @@ func TestHandler_toHTTPStatus(t *testing.T) {
 			require.NoError(t, err)
 
 			status := toHTTPStatus(ctx, tc.err, limits)
-			msg := tc.err.Error()
 			assert.Equal(t, tc.expectedHTTPStatus, status)
-			assert.Equal(t, tc.expectedErrorMsg, msg)
```

Comment:
Why are you removing this assertion?

Reply:
Because that assertion made no sense.
I also removed the `expectedErrorMsg` field.
The tested method, `toHTTPStatus()` maps an error to its HTTP status. So there is no need to check the error message.

Reply:
Thanks for the explanation.